packages/@react-stately/collections/docs/collections.mdx

```diff
@@ -0,0 +1,387 @@
+virtualized scrolling for performance with large collections, and more.
+
+There are many ways one could design an API for components like this: JSX children, a prop that accepts a
+list of option objects, as a datasource object with getter functions for each item, etc.. Selection, and 
+other states like disabled, could be passed as options to each item, or as a separate top-level prop.
+Each of these has various tradeoffs. This page describes how we do it in React Spectrum, and covers some
+of these tradeoffs in detail.
+
+## Design goals
+
+Our main design goal was to provide a consistent API across many types of collection components that is
+easy to learn, performant on large collections, and extensible for advanced features. Some of the usecases
+we considered were:
+
+- Static data
+- Dynamic data
+- Async loading
+- Sections/groups of data from a single source
+- Sections from different sources
+- Single and multiple selection (controlled and uncontrolled)
+- Controlled and uncontrolled expanded states for components like trees and accordions
+- Marking items as selected, expanded, disabled, etc. prior to loading them from a server
+- Drag and drop of items
+- Virtualization for large collections
+- Infinite scrolling to load more items on demand
+
+## Static collections
+
+React Spectrum implements a JSX-based API for defining collections. This allows an intuitive way to provide
+items, which can contain rich rendering of item contents, and various options as props. Building heirarchical
+collections, e.g. sections, or a tree of items is also very natural in JSX.
+
+A **static collection** is a collection that does not change over time (e.g. hard coded). This is common for
+components like action menus where the items are built into the application rather than representing user data.
+
+A simple static collection could be constructed as in the following example.
+
+```tsx
+<Menu>
+  <Item>Open</Item>
+  <Item>Edit</Item>
+  <Item>Delete</Item>
+</Menu>
+```
+
+### Sections
+
+Sections or groups of items can be constructed by wrapping the items as needed.
+
+```tsx
+<ListBox>
+  <Section title="People">
+    <Item>David</Item>
+    <Item>Sam</Item>
+    <Item>Jane</Item>
+  </Section>
+  <Section title="Animals">
+    <Item>Aardvark</Item>
+    <Item>Kangaroo</Item>
+    <Item>Snake</Item>
+  </Section>
+</ListBox>
+```
+
+The &lt;<TypeLink links={docs.links} type={docs.exports.Item} />&gt; and 
+&lt;<TypeLink links={docs.links} type={docs.exports.Section} />&gt; components
+are shared between many collection components. This ensures that they have a consistent
+interface that can be learned once and applied everywhere.
+
+&lt;<TypeLink links={docs.links} type={docs.exports.Item} />&gt; and 
+&lt;<TypeLink links={docs.links} type={docs.exports.Section} />&gt; only define the data for 
+the items and sections, not the rendering, visual appearance, or behavior. This is up to 
+the parent component (e.g. Menu or ListBox).
```

Comment:
*up to the parent component (e.g. Menu or Listbox) to control/define/handle
Otherwise the sentence seems too short or seems like it should be part of the previous sentence.